### DISCUSSION TITLE
More customizable mods

### DISCUSSION THREAD

**Discussion by Resonance2806** [ppy/osu#14138] [Ideas] [answered]
I checked some of the new mods in osu!lazer and I was very surprised because several mods are customizable, but some of them like Hardrock or Easy has little or don't have customizable feature, it's could be better if you add more customizable for mods, especially these two
it's also pretty awesome it's changing the mods feature also change the score multiplier because it's kinda useless if DT speed is reduced to 1.01 (almost no different from no DT) and the score multiplier is still 1.12 (oh yeah, beatmap star too)

**Comment by bdach** [comment, marked as answer]
There is not much point in adding customisation of Hard Rock and Easy. Any customisation functions of those could be largely replaced with Mirror and Difficulty Adjust.

The multiplier figures not being affected by settings is a known issue but it's not a huge priority, as the concept of "mod multipliers" is on the chopping block so to speak in general, as far as I understand.
